expression/builtin_math_test.go

```diff
@@ -115,32 +116,65 @@ func (s *testEvaluatorSuite) TestFloor(c *C) {
 
 	sc := s.ctx.GetSessionVars().StmtCtx
 	tmpIT := sc.IgnoreTruncate
-	sc.IgnoreTruncate = true
```

Comment:
If we keep this line here, the following statement will be overrided by `sc.IgnoreTruncate = test.ignoreTruncate`@L158. In my option, it seems that it's not necessary to keep that line.